api/Navigator.json

```diff
@@ -3132,6 +3132,54 @@
           }
         }
       },
+      "storage": {
+        "__compat": {
```

Comment:
Can you link https://developer.mozilla.org/en-US/docs/Web/API/Navigator/storage?